rasa_addons/core/tracker_stores/botfront.py

```diff
@@ -168,11 +171,13 @@ def _store_tracker_info(self, sender_id, tracker_info):
             }
 
     def save(self, canonical_tracker):
-
-        # Fetch here just in case retrieve wasn't called first
+        serialized_tracker = self._serialize_tracker_to_dict(canonical_tracker)
         sender_id = canonical_tracker.sender_id
+        if self.botfront_test_regex.match(sender_id):
+            self.test_trackers[sender_id] = canonical_tracker
+            return serialized_tracker
```

Comment:
Don,t know if this matters, but you're not returning the same thing as below, ie. `serialized_tracker["events"]`